2-Draft-Accepted/RFCxxxx-DSC-for-PowerShell7.md

```diff
@@ -0,0 +1,323 @@
+---
+RFC: RFCxxxx
+Author: Steve Lee
+Status: Draft
+SupercededBy: N/A
+Version: 1.0
+Area: Desired State Configuration
+Comments Due: Nov 22, 2020
+Plan to implement: Yes
+---
+
+# DSC for PowerShell 7
```

Comment:
Do we need to make it clear that it's not "any 7" per se, but `7.x` (not being 7.0 or 7.1)?  "for PowerShell 7" is used throughout, and in Docs I'd be worried it might infer for an already existing version of 7 since it's not fully explicit.  If this is the normal titling for the RFCs in PS, just ignore this comment. :)

Reply:
The `PowerShell 7` was deliberate as part of our branding and not committing to a specific version of when this work will be complete